pkg/common/k8s_utils_test.go

```diff
@@ -137,3 +140,241 @@ func TestGetServiceWorkloadSelector(t *testing.T) {
 		t.Error("should have failed to get the service workload selector")
 	}
 }
+
+func TestObjectInfo(t *testing.T) {
+	testCases := []struct {
+		name     string
+		input    client.Object
+		expected string
+	}{
+		{
+			name: "when given Kuadrant Limitador object then return formatted string",
+			input: &v1alpha1.Limitador{
+				TypeMeta: metav1.TypeMeta{
+					Kind: "Limitador",
+				},
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "test-limitador",
+				},
+			},
+			expected: "Limitador/test-limitador",
+		},
+		{
+			name: "when given k8s Pod object then return formatted string",
```

Comment:
How is this test different from previous? Is the idea here to highlight that the function works with kind of Kubernetes resource? If so, I'd say not needed because no finite set of examples will suffice.

Reply:
Good point! The intention of this test was indeed to emphasize that the function works with Kuadrant resources (Limitador as example). However, you're right @guicassolato that providing a finite set of examples may not be sufficient.  I am going to get rid of the redundant one!

Thanks for pointing that out @guicassolato!